File: api-eventos/cache.py
import redis
import os
from dotenv import load_dotenv

load_dotenv()
redis_url = os.getenv("REDIS_URL")
client = redis.Redis.from_url(redis_url)

def cache_get(key):
    return client.get(key)

def cache_set(key, value, ttl=60):
    client.setex(key, ttl, value)

File: api-sensores/cache.js
const redis = require('redis');
const { promisify } = require('util');
const dotenv = require('dotenv');
dotenv.config();

const client = redis.createClient({
  url: process.env.REDIS_URL
});
client.connect().catch(console.error);

async function get(key) {
  return await client.get(key);
}

async function set(key, value, ttlSeconds = 60) {
  await client.setEx(key, ttlSeconds, value);
}

module.exports = { get, set };
